src/themes/store/components/Input.js

```diff
@@ -0,0 +1,43 @@
+const Input = {
+  parts: ['field'],
+  baseStyle: {
+    field: {
+      textTransform: 'uppercase',
+      color: 'brand.gray',
+      fontSize: '14px',
+      fontFamily: 'heading',
+      fontWeight: 'bold',
+      lineHeight: '20px',
+      letterSpacing: 'wider',
```

Comment:
These are the styles for the Subtitle `Heading` - I wasn't able to find a clean way to use it. One idea was to import `Heading` and do something like this but if that style ever changes, this will change as well. Any ideas?

```suggestion
      fontFamily: 'heading',
      fontWeight: 'bold',
      ...Heading.sizes.subtitle,
```

Reply:
i think it's fine